### ISSUE TITLE
Cannot find module 'browser-sync'  because npm ERR! shasum check failed

### ISSUE THREAD

**Issue by ptiiCarlos** [npm/npm#7828]
Thank you. Taking John Papa's Pluralsight course AngularJS Patterns: Clean Code, followed along, and tried installing the packages he has in a modular application in his ng-demos on github, and got this error, preventing me from following with the course. Because it says to come here and present the issue, I am doing just that. Thanks again in advance.

npm ERR! shasum check failed for so and so as it expected
```
npm ERR! shasum check failed for C:\Users\Carlos\AppData\Local\Temp\npm-7724-7ac1795b\registry.npmjs.org\node-notifier\-\node-notifier-3.4.1.tgz
npm ERR! Expected: ec0ba696bd6097aaf7c6c3e58b23c849f334bda1
npm ERR! Actual:   aaef6025a82bf7625e1f78b947afa22f18c68801
npm ERR! From:     https://registry.npmjs.org/node-notifier/-/node-notifier-3.4.1.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\play\ng-demos\modular\npm-debug.log
```
This file is very large and am unable to attach it as it can only be png, gif, or jpg, it said the body is too long. I can be reached at carlos@ptii.org and send it to you if needed.

**Comment by othiym23** [comment]
If you could upload `npm-debug.log` to gist.github.com and drop the link here, it would be helpful, but for starters, what versions of npm and node are you running? I double-checked right now, and I was able to install `node-notifier@3.4.1` with no difficulty.

**Comment by ptiiCarlos** [comment]
Hi, thanks so much. npm 2.7.4, I was also able to install node-notifier@3.4.1. I think my error has to do with a previous error that said Karma was expecting version so and so, and I had a newer version. New to nodeJS, bought WebStorm 2 days ago to try to follow John on his course. Can see the code but cannot run anything, the server is not listening as it should on port 1700. Did edit to tell it where node was and what js to use to start up but nothing. I will find a way to upload the file to git and add a link here once I get that done (not sure how).

Thank you very much
Carlos

**Comment by ptiiCarlos** [comment]
I researched a bit more and saw that when I first tried to npm install -g bower gulp karma-cli nodemon, it issued a warning stating karma-cli@0.0.4 wanted node 0.8 or 0.10 and the current version of node I have is  0.12.2, it also mentions npm 2.7.4. I retried to npm install, noticed the packages was not on the c drive, changed directory and retried where the file existed and got it to run and install all it needed. I now  have other issues as in it says angular is not there but I know it is. I will close this issue as it has been resolved. Thank you once again.
Carlos
